ci/release/changelogs/next.md

```diff
@@ -4,4 +4,6 @@
 
 #### Improvements 🧹
 
+- prevent `tooltip` to be an URL when `link` is already set. [#888](https://github.com/terrastruct/d2/pull/1091)
```

Comment:
number is not right

Reply:
oh i see, you used the issue number and linked the PR. they should be the same number though, choose whichever one has more context